src/ramses_rf/device/hvac.py

```diff
@@ -289,16 +289,18 @@ class HvacDisplayRemote(HvacRemote):  # DIS
     #     )
 
 
-class HvacVentilator(FilterChange):  # FAN: RP/31DA, I/31D[9A]
+class HvacVentilator(FilterChange):  # FAN: RP/31DA, I/31D[9A], I/12A0
```

Comment:
No.

Anything that sends `RP/31DA, I/31D[9A]` is - by definition, a FAN.

Devices that send `I/12A0` may not be FANs.